### PR TITLE
Hack 652 - https://github.com/zio/zio-sql/issues/652

### PR DESCRIPTION
**Pull request by AdityaVetukuri**
https://github.com/zio/zio-sql/issues/652

### CODE REVIEW

**Comment by jczuchnowski** [comment]
@AdityaVetukuri it looks good codewise, but you need to:

File: mysql/src/main/scala/zio/sql/mysql/MysqlSqlModule.scala [hunk 1]
```diff
@@ -27,12 +27,13 @@ trait MysqlSqlModule extends Sql { self =>
...
-    val Log10     = FunctionDef[Double, Double](FunctionName("log10"))
-    val Pi        = Expr.FunctionCall0[Double](FunctionDef[Any, Double](FunctionName("pi")))
-    val BitLength = FunctionDef[String, Int](FunctionName("bit_length"))
+    val Crc32       = FunctionDef[String, Long](FunctionName("crc32"))
+    val Degrees     = FunctionDef[Double, Double](FunctionName("degrees"))
+    val Log2        = FunctionDef[Double, Double](FunctionName("log2"))
+    val Log10       = FunctionDef[Double, Double](FunctionName("log10"))
+    val Pi          = Expr.FunctionCall0[Double](FunctionDef[Any, Double](FunctionName("pi")))
+    val BitLength   = FunctionDef[String, Int](FunctionName("bit_length"))
+    val CurrentDate = Expr.ParenlessFunctionCall0[LocalDate](FunctionName("current_date"))
```
Comment (jczuchnowski) [inline mysql/src/main/scala/zio/sql/mysql/MysqlSqlModule.scala:36]: Could you please place the CurrentDate alphabetically after Crc32. And along the way BitLength at the begining?

**Comment by jczuchnowski** [comment]
@AdityaVetukuri just one more small change.

File: mysql/src/main/scala/zio/sql/mysql/MysqlSqlModule.scala (hunk 1 above)
Comment (AdityaVetukuri) [reply mysql/src/main/scala/zio/sql/mysql/MysqlSqlModule.scala:36]: Sure. Thanks for the feedback!

**Comment by AdityaVetukuri** [comment]
Thanks, I have signed the CLA too.

**Review by jczuchnowski — APPROVED** [review]

**Pull request merged**